docs/Privacy/Start-Pantheon-Privacy.md

```diff
@@ -0,0 +1,334 @@
+
+```bash tab="MacOS"
+pantheon --data-path=Node-2-data-path --genesis-file=../privateNetworkGenesis.json 
+--bootnodes="enode://<node public key ex 0x>@127.0.0.1:30303" --p2p-port=30304  
+ --privacy-enabled=true --privacy-precompiled-address=125
+ --privacy-url=127.0.0.1 --rpc-http-api=EEA  
+```
+
+```bash tab="Windows"
+pantheon --data-path=Node-2-data-path --genesis-file=..\privateNetworkGenesis.json --bootnodes="enode://<node public key ex 0x>@127.0.0.1:30303" --p2p-port=30304      
+```
+
+The command line specifies: 
+
+* Different port to Node-1 for P2P peer discovery using the [`--p2p-port`](../Reference/Pantheon-CLI-Syntax.md#p2p-port) option.
+* Enode URL for Node-1 using the [`--bootnodes`](../Reference/Pantheon-CLI-Syntax.md#bootnodes) option.
+* Data directory for Node-2 using the [`--data-path`](../Reference/Pantheon-CLI-Syntax.md#data-path) option.
+* Genesis file as for Node-1.confirm-private-network-is-working
+* [`--privacy-enabled=true`](../Reference/Pantheon-CLI-Syntax.md#privacy-enabled)
+  set true to enable privacy.
+* Changes the default Privacy PreCompiled address
+  [`--privacy-precompiled-address=125`](../Reference/Pantheon-CLI-Syntax.md#privacy-precompiled-address)
+* Setup the enclave(orion) URL
+  [`--privacy-url=127.0.0.1`](../Reference/Pantheon-CLI-Syntax.md#privacy-url)
+* Enable EEA methods
+  [`--rpc-http-api=EEA`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-api)  
+
+### 6. Start Node-3
+
+Start another terminal, change to the `Node-3` directory and start Node-3 replacing the enode URL with your bootnode: 
+
+```bash tab="MacOS"
+pantheon --data-path=Node-3-data-path --genesis-file=../privateNetworkGenesis.json 
+--bootnodes="enode://<node public key ex 0x>@127.0.0.1:30303" --p2p-port30305
+```
+
+```bash tab="Windows"
+pantheon --data-path=Node-3-data-path --genesis-file=..\privateNetworkGenesis.json 
+--bootnodes="enode://<node public key ex 0x>@127.0.0.1:30303" --p2p-port=30305    
+```
+
+The command line specifies: 
+
+ * Different port to Node-1 and Node-2 for P2P peer discovery.
+ * Data directory for Node-3 using the [`--data-path`](../Reference/Pantheon-CLI-Syntax.md#data-path) option.
+ * Bootnode and genesis file as for Node-2. 
+*  Without privacy commandline.
+
+### 7. Confirm Private Network is Working 
+
+Start another terminal, use curl to call the JSON-RPC API [`net_peerCount`](../Reference/JSON-RPC-API-Methods.md#net_peercount) method and confirm the nodes are functioning as peers: 
+
+```bash
+curl -X POST --data '{"jsonrpc":"2.0","method":"net_peerCount","params":[],"id":1}' localhost:8545
+```
+
+The result confirms Node-1 (the node running the JSON-RPC service) has two peers (Node-2 and Node-3):
+```json
+{
+  "jsonrpc" : "2.0",
+  "id" : 1,
+  "result" : "0x2"
+}
+```
+ 
+
+### 8. Create a Private Transaction Node-1 with Node-2
+
+
+
+### 9. Confirm Node-3 can't interact with private Transaction
+
+
```

Comment:
Content still needed describing how to do this?

Reply:
in progress